vignettes/comparison.Rmd

```diff
@@ -0,0 +1,322 @@
+---
+title: Exemplary comparison of Bayesian and frequentist MMRM fits
+date: "`r format(Sys.time(), '%d/%m/%Y')`"
+output:
+  rmarkdown::html_document:
+    theme: spacelab
+    highlight: default
+    toc: yes
+    toc_float: yes
+    number_sections: true 
+vignette: >
+  %\VignetteIndexEntry{Exemplary comparison of Bayesian and frequentist MMRM fits}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+editor_options:
+  chunk_output_type: console
+  markdown:
+    wrap: 72
+---
+
+```{r setup, eval=TRUE, include=FALSE, warning=FALSE, message=FALSE}
+rm(list = ls())
+library(knitr)
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  warning = FALSE,
+  message = FALSE,
+  prompt = TRUE,
+  comment = NA,
+  size = "scriptsize",
+  fig.height = 5,
+  fig.width = 6,
+  dpi = 320,
+  fig.align = "center",
+  fig.path = "figures/"
+)
+```
+
+# About {.unnumbered}
+
+This vignette provides an exemplary comparison of a Bayesian MMRM fit, 
+obtained by `brms.mmrm::brm_model()`, and a frequentist MMRM fit, 
+obtained by `mmrm::mmrm()`.
+
+# Prerequisites
+
+Load required packages:
+
+```{r load_gen_pkgs, eval=TRUE, echo=TRUE, class.source = NULL}
+packages <- c(
+  "dplyr",
+  "tidyr",
+  "ggplot2",
+  "gt",
+  "gtsummary",
+  "purrr",
+  "parallel",
+  "brms.mmrm",
+  "mmrm",
+  "emmeans",
+  "posterior"
+)
+invisible(lapply(packages, library, character.only = TRUE))
+```
+
+Set seed:
+```{r set_seed, eval=TRUE, echo=TRUE, class.source = NULL}
+set.seed(123L)
+```
+
+# Data
+
+## Pre-processing
+
+The `fev_dat` dataset is used that is contained in `{mmrm}`:
```

Comment:
Might be nice to include a short description of the dataset, e.g. the "Note" in https://openpharma.github.io/mmrm/latest-tag/reference/fev_data.html.

Also, I see on that page:

> This is an artificial dataset.

Would be nice to use a real clinical dataset in addition.

Reply:
I agree. Other datasets in the end did not convince me as good (educational) examples, and I also wanted to avoid further dependencies. OK for you to leave it with `fev_data` for now and add or switch to another dataset later, perhaps one from `clindata`?

Reply:
A short description has now been added (1a28cb778aa22671c78705a2b937601a095918ac). OK to add or switch to another dataset later, perhaps one from the `clindata`-package? The current alternatives weren't convincing for me as good (educational) examples and I also wanted to avoid further dependencies, this is why I sticked to the `mmrm`-package.

Reply:
Thanks @chstock. I like your idea to align with the `mmrm` package. Indeed, I see a lot of work at https://openpharma.github.io/mmrm/latest-tag/articles/mmrm_review_methods.html which uses both the FEV and BCVA datasets. In the end, I think it would be sufficient to analyze both and refer readers to the results from https://openpharma.github.io/mmrm/latest-tag/articles/mmrm_review_methods.html#marginal-treatment-effect-estimates-comparison. But that could be a future PR.